test/component/deceased/testDetails.js

```diff
@@ -12,14 +16,28 @@ describe('deceased-details', () => {
 
     beforeEach(() => {
         testWrapper = new TestWrapper('DeceasedDetails');
+        nock(featureToggleUrl)
+            .get(featureTogglePath)
+            .reply(200, 'true');
     });
 
     afterEach(() => {
         testWrapper.destroy();
+        nock.cleanAll();
     });
 
     describe('Verify Content, Errors and Redirection', () => {
-        testHelpBlockContent.runTest('DeceasedDetails');
+        it('test help block content is loaded on page', (done) => {
+            const playbackData = {};
+            playbackData.helpTitle = commonContent.helpTitle;
+            playbackData.helpText = commonContent.helpText;
+            playbackData.contactTelLabel = commonContent.contactTelLabel.replace('{helpLineNumber}', config.helpline.number);
+            playbackData.contactOpeningTimes = commonContent.contactOpeningTimes.replace('{openingTimes}', config.helpline.hours);
+            playbackData.helpEmailLabel = commonContent.helpEmailLabel;
+            playbackData.contactEmailAddress = commonContent.contactEmailAddress;
+
+            testWrapper.testDataPlayback(done, playbackData);
+        });
```

Comment:
If we change `testHelpBlockContent.js` so it accepts a callback param as in https://github.com/hmcts/probate-frontend/blob/PRI-114-Create-Intestacy-Services/test/component/common/testHelpBlockContent.js then we can pass in `nock()` and use the generic test rather than duplicating it in this test.